src/sites/cclw/constants/faqs.tsx

```diff
@@ -90,29 +103,30 @@ export const FAQS: TFAQ[] = [
     ),
   },
   {
-    title: "Am I free to download and use your data?",
+    title: "Am I free to download and use the data?",
     content: (
       <>
         <p>
           Yes - and we encourage you to do so! The content of our database is available under the Creative Commons Attribution Licence{" "}
           <ExternalLink url="https://creativecommons.org/licenses/by/4.0/">(CC-BY)</ExternalLink>. Before doing so, you should read our Terms of Use
-          for more information and to find out how to credit our data. You will also need to{" "}
+          for more information and to find out how to cite and credit the resources. If you wish to download the data as a .csv file, please{" "}
```

Comment:
I think we should point out that the terms of use differs for various sources here /cc @henry-cpr

Reply:
Yes  I think you're right

Reply:
Raised  https://linear.app/climate-policy-radar/issue/SOF-55/review-the-use-of-terms-and-conditions-of-third-party-sites